.travis.yml

```diff
@@ -49,13 +49,9 @@ addons:
       - doxygen-latex
       - dvipng
       - libusb-1.0-0-dev
-cache:
-  directories:
-  - $HOME/flann
-  - $HOME/vtk
-  - $HOME/qhull
-  - $HOME/doxygen
-before_install:
-  - bash .travis.sh install
+      - libqhull-dev
+      - libvtk5-dev
+      - libflann-dev
+      - doxygen
```

Comment:
[This](https://launchpad.net/~kedazo/+archive/ubuntu/doxygen-updates-precise) looks exactly like the solution we need. Doxygen packages backported for Precise. It will bump the version to 1.8.8 which already has markdown support.